Rmarkdown/Oncology_Efficacy_Plots.Rmd

```diff
@@ -254,29 +304,31 @@ When two drugs are combined (ABC123 + DEF456), it can be a useful first step to
 gg <- ggplot()
 gg <- gg + geom_point(data=my.data.monotherapy,aes(x=DOSE_ABC,y=BPCHG, color="ABC123"))
 gg <- gg + geom_point(data=my.data.combo,aes(x=DOSE_ABC,y=BPCHG, 
-                                          color = factor(DOSE_DEF, labels = c("ABC123 + 300 mg DEF456", "ABC123 + 400 mg DEF456"))))
+                                          color = factor(DOSE_DEF, labels = c("ABC123 + 400 mg DEF456", "ABC123 + 300 mg DEF456"))))
```

Comment:
should we add DOSE_DEF_label and DOSE_ABC_label definitions to the top of the page?

Reply:
similar to DOSE_label_low2high and DOSE_label_high2low?

Reply:
Maybe name them MONO_DOSE_label_low2high and COMBO_DOSE_label_low2high?

Reply:
done!